### DISCUSSION TITLE
a question about the moordyn

### DISCUSSION THREAD

**Discussion by wufangyun92** [OpenFAST/openfast#2529] [Q&A]
Originally, it was possible to run Moordyn at the beginning, but when I changed Tmax to 300 seconds, he couldn't run anymore and could only run with MAP++. I want to ask how to solve this problem.
![image](https://github.com/user-attachments/assets/8e20f0d6-3720-42c4-b92f-ded487fc625d)

**Comment by RyanDavies19** [comment]
Hi @wufangyun92, have you updated your MoorDyn input file format to version 2? See https://moordyn.readthedocs.io/en/latest/inputs.html#the-v2-input-file for the format. 

The issue returned by moordyn: "MD_INIT: Failed to read options" means that there is an issue with the input file.
